shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DDLStatement.g4

```diff
@@ -130,6 +130,10 @@ dropDatabase
     : DROP DATABASE existClause? name
     ;
 
+dropGroup
+    : DROP GROUP existClause? name
```

Comment:
Note that multiple groups can be dropped at one time
![image](https://user-images.githubusercontent.com/66914151/160299409-918baf9f-c6b5-45ab-b5a3-891503a644c0.png)
you can achieve this by `name (COMMA_ name)*`

Reply:
Made the changes